Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/ComponentConfig.purs

```diff
@@ -667,6 +667,12 @@ yatriRateCardList vehicleVariant state = do
                    , { key : if lang == "EN_US" then (getString MORE_THAN) <> " 30 km" else "30 " <> (getString MORE_THAN) , val :"₹40 / km"}
                    , { key : (getString PICKUP_CHARGE), val : "₹" <> (show state.data.pickUpCharges) }
                    , { key : (getString DRIVER_ADDITIONS) ,val : "₹0 - ₹60"}]
+
+    "AUTO_RICKSHAW"  -> [ { key : if lang == "EN_US" then (getString MIN_FARE_UPTO) <> " 1.5 km" else "1.5 km " <> (getString MIN_FARE_UPTO) , val : "₹30"}
+                        , { key : if lang == "EN_US" then (getString RATE_ABOVE_MIN_FARE) <> "15 / km" else (getString RATE_ABOVE_MIN_FARE), val : "₹15 / km"}
```

Comment:
change `if lang == "EN_US" then (getString RATE_ABOVE_MIN_FARE) <> "15 / km" else (getString RATE_ABOVE_MIN_FARE)` to `(getString RATE_ABOVE_MIN_FARE)`

Reply:
done